config/deploy.yml

```
@@ -0,0 +1,39 @@
+env:
+  secret:
+    - API_BASE_URL
+    
+proxy:
+  ssl: false               
+  hosts:
+    - camp-sync.polo-apps.com
+  app_port: 9000 
+
+registry:
+  server: ghcr.io          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)